### PR TITLE
signal: add page

### PR DESCRIPTION
**Pull request by munntjlx**
This is mostly a reprint from the man signal (which lists the kill signal types)

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/signal.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     23    SIGIO            discard signal       I/O is possible on a descriptor (see fcntl(2))
+     24    SIGXCPU          terminate process    cpu time limit exceeded (see setrlimit(2))
+     25    SIGXFSZ          terminate process    file size limit exceeded (see setrlimit(2))
+     26    SIGVTALRM        terminate process    virtual time alarm (see setitimer(2))
+     27    SIGPROF          terminate process    profiling timer alarm (see setitimer(2))
+     28    SIGWINCH         discard signal       Window size change
+     29    SIGINFO          discard signal       status request from keyboard
+     30    SIGUSR1          terminate process    User defined signal 1
+     31    SIGUSR2          terminate process    User defined signal 2
+```
```
Comment (kbdharun) [inline pages/osx/signal.md:42]:
```suggestion
> Simplified software signal facilities.
> More information: <https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/signal.3.html>.

- View documentation for the command:

`man signal`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. Unfortunately, our style guide doesn't allow for these kinds of lists. 

An alternative is to link to the man page and add the command to check it locally. I have done the same in my suggestion below.

**Review by kbdharun — APPROVED** [review]

**Review by munntjlx — COMMENTED** [review]
Looks great!

**Review by acuteenvy — CHANGES_REQUESTED** [review]
`signal` is not a command and the linked man page just lists the signals that can be sent to processes via other means.
I don't think there's any other page documenting something that isn't a command.

> \- View documentation for the `signal` command:

If something like this ***is*** allowed, then that line should probably be reworded, as there is no `signal` command.

**Comment by kbdharun** [comment]
> `signal` is not a command and the linked man page just lists the signals that can be sent to processes via other means.
> I don't think there's any other page documenting something that isn't a command.
> 
> > \- View documentation for the `signal` command:
> 
> If something like this ***is*** allowed, then that line should probably be reworded, as there is no `signal` command.

Yeah, we do have an exception of non command pages for `osx` (Macos) for non invokable daemons, etc. We can reword the the man page description here.

File: pages/osx/signal.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# signal
+
+> Simplified software signal facilities.
+> More information: <https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/signal.3.html>.
+
+- View documentation for the `signal` command:
```
Comment (kbdharun) [inline pages/osx/signal.md:6]:
```suggestion
- View documentation for the list of signals in Mac OS:
```

Comment (acuteenvy) [inline pages/osx/signal.md:6]:
```suggestion
- View documentation about signals in Mac OS:
```

**Comment by acuteenvy** [comment]
Merge whichever suggestion you think is better, I was making one at the same time as you.

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**